blocks/editable/format-toolbar/index.js

```diff
@@ -168,12 +172,14 @@ class FormatToolbar extends Component {
 		);
 
 		return (
-			<div className="blocks-format-toolbar">
+			<div className="blocks-format-toolbar" >
 				<Toolbar controls={ toolbarControls } />
 
 				{ ( isAddingLink || isEditingLink ) &&
+					/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
Disabling a rule will impact the rest of the file. We should make sure to re-enable it when we expect that it should be take effect again.

I believe it is also possible to make it more specific to the form you're considering via the `{` JSX syntax for inserting JavaScript, e.g.

```jsx
<Fill name="Editable.Siblings">
	{ /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */ }
	<form onMouseDown={ this.stopMousePropagation } />
	{ /* eslint-enable jsx-a11y/no-noninteractive-element-interactions */ }
</Fill>
```

Finally, to discourage unwarranted rule disabling, we've made a point to establish a convention for including a comment explaining why the rule should be disabled (even for our benefit of understanding the rule, when and where it shouldn't apply).

Reply:
Understood